AppCenter/AppCenter/Internals/Context/AuthToken/MSAuthTokenContext.h

```diff
@@ -0,0 +1,28 @@
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@protocol MSAuthTokenContextDelegate;
```

Comment:
Consider using internal header for methods only used internally.

Reply:
It's internal already since part of the "Internals" folder. When there is no "public" header for the class we don't bother explicitly mark the header filename with "Internal".